typescript/infra/config/registry.ts

```diff
@@ -7,7 +7,10 @@ import {
   PartialRegistry,
   warpConfigToWarpAddresses,
 } from '@hyperlane-xyz/registry';
-import { FileSystemRegistry } from '@hyperlane-xyz/registry/fs';
+import {
+  FileSystemRegistry,
+  getRegistry as getMergedRegistry,
```

Comment:
Why call this merged registry? Just call this registry, to the caller, it shouldn't matter if its a mergedRegistry no?

Reply:
there's an existing getRegistry